app/assets/stylesheets/govuk_publishing_components/components/_intervention.scss

```diff
@@ -0,0 +1,24 @@
+.gem-c-intervention {
+  @include govuk-text-colour;
+  @include govuk-responsive-padding(4);
+  @include govuk-responsive-margin(4, "bottom");
+
+  background-color: govuk-colour("light-grey", $legacy: "grey-4");
+}
+
+.gem-c-intervention__title {
+  @include govuk-font(24, $weight: bold);
+  margin-top: 0;
+  @include govuk-responsive-margin(4, "bottom");
+
+  a {
+    @include govuk-link-common;
+    @include govuk-link-style-default;
+    @include govuk-link-print-friendly;
+  }
+}
+
+.gem-c-intervention__description,
+.gem-c-intervention__dismiss {
```

Comment:
If these two classes are styled exactly the same, could they be merged into a single class?

Reply:
I'm sure they'll diverge eventually but makes sense not to anticipate things too much